### ISSUE TITLE
Gradient Accumulation for training large batch size on single GPU

### ISSUE THREAD

**Issue by karanchahal** [facebookresearch/maskrcnn-benchmark#765]
Can we have a feature that allows gradient accumulation so that we can train on a larger batch size on a single GPU, much more than the GPU memory allows? 

[This](https://medium.com/huggingface/training-larger-batches-practical-tips-on-1-gpu-multi-gpu-distributed-setups-ec88c3e51255) article tells us how to do it.  This would be a really useful feature as I understand one cannot get the same types of state of the art accuracies with a smaller batch size.
